src/components/detail/ContentInfo.vue

```diff
@@ -34,7 +48,7 @@ const shareOnTwitter = () => {
     .open(
       `https://twitter.com/intent/tweet?text=${encodeURIComponent(
         `"${props.title.title}" by ${props.title.authorName} #JankariTech`
```

Comment:
Looks like there is separate param for hashtags. See https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview
```suggestion
        `"${props.title.title}" by ${props.title.authorName}&hashtags=JankariTech`
```